app/views/IndexView.scala.html

```diff
@@ -39,33 +42,81 @@
     fullWidth = true
 ) {
 
-    @if(paymentError) {
-        <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
-            <div class="govuk-notification-banner__header">
-                <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
-                @messages("notificationBanner.title")
-                </h2>
-            </div>
-            <div class="govuk-notification-banner__content">
-                <p class="govuk-notification-banner__heading">
-                @messages("notificationBanner.text")
-                </p>
-            </div>
-        </div>
-    }
+@if(paymentError) {
+<div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title"
+     data-module="govuk-notification-banner">
+    <div class="govuk-notification-banner__header">
+        <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+            @messages("notificationBanner.title")
+        </h2>
+    </div>
+    <div class="govuk-notification-banner__content">
+        <p class="govuk-notification-banner__heading">
+            @messages("notificationBanner.text")
+        </p>
+    </div>
+</div>
+}
+
+<h1 class="govuk-heading-l">
+    @messages("index.heading")
+    <span class="govuk-caption-m">@businessName</span>
+    <span class="govuk-caption-m">@messages("index.vrn", vrn)</span>
+</h1>
+
+@if(exclusionsEnabled) {
+    <div class="govuk-warning-text">
+        <strong class="govuk-warning-text__text">
+            <span class="govuk-warning-text__assistive">Warning</span>
```

Comment:
I'd like a demo of this page, with the following scenario:

- exclusions enabled
- user is not excluded

I feel like this part will still be shown